src/overmap.cpp

```diff
@@ -1348,139 +1385,95 @@ void overmap::draw(WINDOW *w, WINDOW *wbar, const tripoint &center,
         }
     }
 
-    for (int i = 0; i < om_map_width; i++) {
-        for (int j = 0; j < om_map_height; j++) {
-            const int omx = cursx + i - (om_map_width / 2);
-            const int omy = cursy + j - (om_map_height / 2);
+    // A small LRU cache: most oter_id's occur in clumps like forests of swamps.
```

Comment:
How'd this do in benchmarking?
You did test performance with and without it, right?